README.md

```diff
@@ -72,6 +147,93 @@ any draft still accepting bug fixes, please
 [file an issue](https://github.com/json-schema-org/JSON-Schema-Test-Suite/issues)
 or [submit a PR](https://github.com/json-schema-org/JSON-Schema-Test-Suite).
 
+## How to Utilize the Tests
+
+The test suite structure was described
+[above](#introduction-to-the-test-suite-structure). This section describes:
+
+1. How to implement a test runner for testing a validator implementation.
+2. Assumptions the test suite makes and their rationales.
+
+Note that the specific steps described here outline a procedure for running the
+tests. The procedure doesn't need to be followed exactly, but the results of
+your own procedure should produce the same effects.
+
+After completing the tests, each test is marked as one of:
+1. Pass
+2. Fail
+3. Not Executed
+
+### How to Implement a Test Runner
+
+To test a specific draft, walk the filesystem tree for that draft and execute
+all the tests in each `.json` file encountered. Each `.json` file is either
+located in the root of the draft test hierarchy or in a subdirectory. The
+approach is similar for both cases, but tests found in subdirectories need to
+follow the assumptions and restrictions for the containing subdirectory.
+
+For each _test case_ in a `.json` file:
+1. Load the schema from `"schema"`.
+   1. If the schema is loaded successfully then it can be used for each test.
+   2. If the schema is not loaded successfully or is found to be invalid, all
+      _tests_ in this _test case_ should be marked as "Not Executed". It is not
+      correct to assume that `"valid"` is false for these tests.
```

Comment:
This is not correct. We do not supply schemas that are malformed or invalid in the test suite.